### ISSUE TITLE
How to fitune in my own dataset 

### ISSUE THREAD

**Issue by Flyooofly** [ShiqiYu/OpenGait#65]
## System information (version)

- Pytorch => :grey_question:
- Operating System / Platform => :grey_question:
- Cuda => :grey_question:

## Detailed description
fitune in my own dataset ,becausee the accuracy in my dataset is bad that using the public dataset weight

## Issue submission checklist
 - [ ] I checked the problem with documentation, FAQ, issues, and have not found solution.

**Comment by darkliang** [comment]
Hi，you can set the restore_hint to load the pretrained model weights. More details can be found in https://github.com/ShiqiYu/OpenGait/blob/715e7448fa237cc050ca0e31efb2c314edf849ef/docs/1.detailed_config.md#L59-L69
